hadoop/src/tesser/hadoop.clj

```diff
@@ -200,3 +200,68 @@
                   first)]
         (when (error? x) (throw (ex-info "Hadoop fold error" x)))
         x))))
+
+(defn identity-mapper
+  "Does nothing in the map step."
+  {::mr/source-as :vals
+   ::mr/sink-as   :vals}
+  [records]
+  (r/map identity records))
```

Comment:
I thought I tried that and it didn't work. Just tried it again and it worked, so I changed it.